setup.cfg

```diff
@@ -8,6 +8,7 @@ packages =
     flocx_market
 
 [entry_points]
+oslo.config.opts =
+    flocx-market.conf = flocx_market.conf.opts:list_opts
 console_scripts =
-esi_leap.database.migration_backend = 
-	#sqlalchemy = esi_leap.db.sqlalchemy.migration
+    flocx-market-api = flocx_market.cmd.api:main
```

Comment:
The console_scripts entrypoint does not appear to be related to the configuration changes.